2022-01-18 00:00:00.004 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :32   [32m3345649261219840  [0;39m 生成今日电子书快照开始
2022-01-18 00:00:00.363 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :35   [32m3345649261219840  [0;39m 生成今日电子书快照结束，耗时：359毫秒
2022-01-18 00:00:05.016 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3345670283071488  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:00:05.016 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3345670283071488  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:00:15.169 INFO  c.a.insect.knowledgebase.websocket.WebSocketServer:41   [32m                  [0;39m 连接关闭，token：3aKbdgNcc3，session id：b！当前连接数：0
2022-01-18 00:00:18.081 INFO  c.a.insect.knowledgebase.websocket.WebSocketServer:32   [32m                  [0;39m 有新连接：token：KtPZKIe0Hx，session id：c，当前连接数：1
2022-01-18 00:00:35.005 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3345796066054144  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:00:35.005 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3345796066054144  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:01:00.009 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :32   [32m3345900940431360  [0;39m 生成今日电子书快照开始
2022-01-18 00:01:00.016 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :35   [32m3345900940431360  [0;39m 生成今日电子书快照结束，耗时：6毫秒
2022-01-18 00:01:05.008 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3345921907757056  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:01:05.008 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3345921907757056  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:01:33.761 INFO  c.a.insect.knowledgebase.websocket.WebSocketServer:41   [32m                  [0;39m 连接关闭，token：KtPZKIe0Hx，session id：c！当前连接数：0
2022-01-18 00:01:35.014 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3346047762042880  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:01:35.014 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3346047762042880  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:01:35.953 INFO  c.a.insect.knowledgebase.websocket.WebSocketServer:32   [32m                  [0;39m 有新连接：token：PLwyWlNX8I，session id：d，当前连接数：1
2022-01-18 00:01:50.062 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :32   [32m                  [0;39m ------------- LoginInterceptor 开始 -------------
2022-01-18 00:01:50.063 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :43   [32m                  [0;39m 接口登录拦截：，path：http://127.0.0.1:8880/tag/getTagListPage
2022-01-18 00:01:50.063 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :47   [32m                  [0;39m 登录校验开始，token：3334694880022528
2022-01-18 00:01:50.064 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :59   [32m                  [0;39m 已登录：{"id":1,"passWord":"1f15d405949c4ed3c2279aa6d48b1e9a","token":"3334694880022528","userCode":"admin","userName":"李雨泽"}
2022-01-18 00:01:50.067 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :68   [32m                  [0;39m ------------- LoginInterceptor 结束 耗时：4 ms -------------
2022-01-18 00:01:56.616 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :32   [32m                  [0;39m ------------- LoginInterceptor 开始 -------------
2022-01-18 00:01:56.616 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :43   [32m                  [0;39m 接口登录拦截：，path：http://127.0.0.1:8880/tag/getTagListPage
2022-01-18 00:01:56.616 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :47   [32m                  [0;39m 登录校验开始，token：3334694880022528
2022-01-18 00:01:56.618 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :59   [32m                  [0;39m 已登录：{"id":1,"passWord":"1f15d405949c4ed3c2279aa6d48b1e9a","token":"3334694880022528","userCode":"admin","userName":"李雨泽"}
2022-01-18 00:01:56.620 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :68   [32m                  [0;39m ------------- LoginInterceptor 结束 耗时：4 ms -------------
2022-01-18 00:01:59.900 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :32   [32m                  [0;39m ------------- LoginInterceptor 开始 -------------
2022-01-18 00:01:59.900 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :43   [32m                  [0;39m 接口登录拦截：，path：http://127.0.0.1:8880/user/getUserListPage
2022-01-18 00:01:59.901 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :47   [32m                  [0;39m 登录校验开始，token：3334694880022528
2022-01-18 00:01:59.902 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :59   [32m                  [0;39m 已登录：{"id":1,"passWord":"1f15d405949c4ed3c2279aa6d48b1e9a","token":"3334694880022528","userCode":"admin","userName":"李雨泽"}
2022-01-18 00:01:59.905 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :68   [32m                  [0;39m ------------- LoginInterceptor 结束 耗时：5 ms -------------
2022-01-18 00:02:00.010 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :32   [32m3346152602865664  [0;39m 生成今日电子书快照开始
2022-01-18 00:02:00.021 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :35   [32m3346152602865664  [0;39m 生成今日电子书快照结束，耗时：11毫秒
2022-01-18 00:02:05.006 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3346173557608448  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:02:05.006 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3346173557608448  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:02:35.013 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3346299416088576  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:02:35.013 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3346299416088576  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:02:59.999 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :32   [32m3346404214968320  [0;39m 生成今日电子书快照开始
2022-01-18 00:03:00.030 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :35   [32m3346404214968320  [0;39m 生成今日电子书快照结束，耗时：31毫秒
2022-01-18 00:03:05.015 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3346425253597184  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:03:05.015 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3346425253597184  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:03:35.000 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3346551019802624  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:03:35.000 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3346551019802624  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:04:00.016 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :32   [32m3346655944511488  [0;39m 生成今日电子书快照开始
2022-01-18 00:04:00.016 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :35   [32m3346655944511488  [0;39m 生成今日电子书快照结束，耗时：0毫秒
2022-01-18 00:04:05.009 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3346676886671360  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:04:05.009 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3346676886671360  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:04:35.022 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3346802770317312  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:04:35.022 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3346802770317312  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:05:00.003 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :32   [32m3346907548225536  [0;39m 生成今日电子书快照开始
2022-01-18 00:05:00.003 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :35   [32m3346907548225536  [0;39m 生成今日电子书快照结束，耗时：0毫秒
2022-01-18 00:05:05.009 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3346928544911360  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:05:05.009 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3346928544911360  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:05:35.017 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3347054407585792  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:05:35.017 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3347054407585792  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:06:00.009 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :32   [32m3347159231631360  [0;39m 生成今日电子书快照开始
2022-01-18 00:06:00.009 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :35   [32m3347159231631360  [0;39m 生成今日电子书快照结束，耗时：0毫秒
2022-01-18 00:06:05.016 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3347180232511488  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:06:05.016 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3347180232511488  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:06:33.480 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :32   [32m                  [0;39m ------------- LoginInterceptor 开始 -------------
2022-01-18 00:06:33.480 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :43   [32m                  [0;39m 接口登录拦截：，path：http://127.0.0.1:8880/user/getUserListPage
2022-01-18 00:06:33.480 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :47   [32m                  [0;39m 登录校验开始，token：3334694880022528
2022-01-18 00:06:33.495 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :59   [32m                  [0;39m 已登录：{"id":1,"passWord":"1f15d405949c4ed3c2279aa6d48b1e9a","token":"3334694880022528","userCode":"admin","userName":"李雨泽"}
2022-01-18 00:06:33.498 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :68   [32m                  [0;39m ------------- LoginInterceptor 结束 耗时：18 ms -------------
2022-01-18 00:06:35.007 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3347306023882752  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:06:35.007 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3347306023882752  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:06:35.231 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :32   [32m                  [0;39m ------------- LoginInterceptor 开始 -------------
2022-01-18 00:06:35.231 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :43   [32m                  [0;39m 接口登录拦截：，path：http://127.0.0.1:8880/tag/getTagListPage
2022-01-18 00:06:35.231 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :47   [32m                  [0;39m 登录校验开始，token：3334694880022528
2022-01-18 00:06:35.232 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :59   [32m                  [0;39m 已登录：{"id":1,"passWord":"1f15d405949c4ed3c2279aa6d48b1e9a","token":"3334694880022528","userCode":"admin","userName":"李雨泽"}
2022-01-18 00:06:35.235 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :68   [32m                  [0;39m ------------- LoginInterceptor 结束 耗时：4 ms -------------
2022-01-18 00:06:39.990 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :32   [32m                  [0;39m ------------- LoginInterceptor 开始 -------------
2022-01-18 00:06:39.990 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :43   [32m                  [0;39m 接口登录拦截：，path：http://127.0.0.1:8880/tag/getTagListPage
2022-01-18 00:06:39.990 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :47   [32m                  [0;39m 登录校验开始，token：3334694880022528
2022-01-18 00:06:39.992 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :59   [32m                  [0;39m 已登录：{"id":1,"passWord":"1f15d405949c4ed3c2279aa6d48b1e9a","token":"3334694880022528","userCode":"admin","userName":"李雨泽"}
2022-01-18 00:06:39.994 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :68   [32m                  [0;39m ------------- LoginInterceptor 结束 耗时：4 ms -------------
2022-01-18 00:06:48.066 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :32   [32m                  [0;39m ------------- LoginInterceptor 开始 -------------
2022-01-18 00:06:48.066 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :43   [32m                  [0;39m 接口登录拦截：，path：http://127.0.0.1:8880/tag/getTagListPage
2022-01-18 00:06:48.066 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :47   [32m                  [0;39m 登录校验开始，token：3334694880022528
2022-01-18 00:06:48.067 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :59   [32m                  [0;39m 已登录：{"id":1,"passWord":"1f15d405949c4ed3c2279aa6d48b1e9a","token":"3334694880022528","userCode":"admin","userName":"李雨泽"}
2022-01-18 00:06:48.069 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :68   [32m                  [0;39m ------------- LoginInterceptor 结束 耗时：3 ms -------------
2022-01-18 00:06:49.321 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :32   [32m                  [0;39m ------------- LoginInterceptor 开始 -------------
2022-01-18 00:06:49.321 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :43   [32m                  [0;39m 接口登录拦截：，path：http://127.0.0.1:8880/user/getUserListPage
2022-01-18 00:06:49.322 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :47   [32m                  [0;39m 登录校验开始，token：3334694880022528
2022-01-18 00:06:49.323 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :59   [32m                  [0;39m 已登录：{"id":1,"passWord":"1f15d405949c4ed3c2279aa6d48b1e9a","token":"3334694880022528","userCode":"admin","userName":"李雨泽"}
2022-01-18 00:06:49.325 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :68   [32m                  [0;39m ------------- LoginInterceptor 结束 耗时：4 ms -------------
2022-01-18 00:07:00.018 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :32   [32m3347410927620096  [0;39m 生成今日电子书快照开始
2022-01-18 00:07:00.018 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :35   [32m3347410927620096  [0;39m 生成今日电子书快照结束，耗时：0毫秒
2022-01-18 00:07:05.003 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3347431836225536  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:07:05.003 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3347431836225536  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:07:35.014 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3347557711482880  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:07:35.014 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3347557711482880  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:08:00.004 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :32   [32m3347662527139840  [0;39m 生成今日电子书快照开始
2022-01-18 00:08:00.004 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :35   [32m3347662527139840  [0;39m 生成今日电子书快照结束，耗时：0毫秒
2022-01-18 00:08:05.010 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3347683523825664  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:08:05.010 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3347683523825664  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:08:35.009 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3347809348751360  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:08:35.009 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3347809348751360  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:09:00.009 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :32   [32m3347914206351360  [0;39m 生成今日电子书快照开始
2022-01-18 00:09:00.009 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :35   [32m3347914206351360  [0;39m 生成今日电子书快照结束，耗时：0毫秒
2022-01-18 00:09:05.015 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3347935203037184  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:09:05.015 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3347935203037184  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:09:23.965 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :32   [32m                  [0;39m ------------- LoginInterceptor 开始 -------------
2022-01-18 00:09:23.965 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :43   [32m                  [0;39m 接口登录拦截：，path：http://127.0.0.1:8880/tag/getTagListPage
2022-01-18 00:09:23.965 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :47   [32m                  [0;39m 登录校验开始，token：3334694880022528
2022-01-18 00:09:23.966 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :59   [32m                  [0;39m 已登录：{"id":1,"passWord":"1f15d405949c4ed3c2279aa6d48b1e9a","token":"3334694880022528","userCode":"admin","userName":"李雨泽"}
2022-01-18 00:09:23.968 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :68   [32m                  [0;39m ------------- LoginInterceptor 结束 耗时：3 ms -------------
2022-01-18 00:09:26.489 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :32   [32m                  [0;39m ------------- LoginInterceptor 开始 -------------
2022-01-18 00:09:26.490 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :43   [32m                  [0;39m 接口登录拦截：，path：http://127.0.0.1:8880/user/getUserListPage
2022-01-18 00:09:26.490 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :47   [32m                  [0;39m 登录校验开始，token：3334694880022528
2022-01-18 00:09:26.491 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :59   [32m                  [0;39m 已登录：{"id":1,"passWord":"1f15d405949c4ed3c2279aa6d48b1e9a","token":"3334694880022528","userCode":"admin","userName":"李雨泽"}
2022-01-18 00:09:26.494 INFO  c.a.i.knowledgebase.interceptor.LoginInterceptor  :68   [32m                  [0;39m ------------- LoginInterceptor 结束 耗时：4 ms -------------
2022-01-18 00:09:35.012 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3348061019574272  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:09:35.012 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3348061019574272  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 00:09:35.378 INFO  c.a.insect.knowledgebase.websocket.WebSocketServer:41   [32m                  [0;39m 连接关闭，token：PLwyWlNX8I，session id：d！当前连接数：0
2022-01-18 00:10:00.018 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :32   [32m3348165902340096  [0;39m 生成今日电子书快照开始
2022-01-18 00:10:00.018 INFO  c.amber.insect.knowledgebase.job.EbookSnapshotJob :35   [32m3348165902340096  [0;39m 生成今日电子书快照结束，耗时：0毫秒
2022-01-18 00:10:05.003 INFO  com.amber.insect.knowledgebase.job.DocJob         :32   [32m3348186810945536  [0;39m 更新电子书下的文档数据开始
2022-01-18 00:10:05.003 INFO  com.amber.insect.knowledgebase.job.DocJob         :35   [32m3348186810945536  [0;39m 更新电子书下的文档数据结束，耗时：0毫秒
2022-01-18 17:39:03.622 INFO  org.hibernate.validator.internal.util.Version     :21   [32m                  [0;39m HV000001: Hibernate Validator 6.2.0.Final
2022-01-18 17:39:03.647 INFO  c.a.insect.knowledgebase.KnowledgeBaseApplication :55   [32m                  [0;39m Starting KnowledgeBaseApplication using Java 1.8.0_291 on ZB-20210629CYMT with PID 12420 (D:\ext.liyuze\Documents\code\knowledge-base\target\classes started by ext.liyuze in D:\ext.liyuze\Documents\code\knowledge-base)
2022-01-18 17:39:03.648 INFO  c.a.insect.knowledgebase.KnowledgeBaseApplication :635  [32m                  [0;39m No active profile set, falling back to default profiles: default
2022-01-18 17:39:03.728 INFO  o.s.b.d.env.DevToolsPropertyDefaultsPostProcessor :255  [32m                  [0;39m Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2022-01-18 17:39:03.729 INFO  o.s.b.d.env.DevToolsPropertyDefaultsPostProcessor :255  [32m                  [0;39m For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2022-01-18 17:39:06.827 INFO  o.s.d.r.config.RepositoryConfigurationDelegate    :262  [32m                  [0;39m Multiple Spring Data modules found, entering strict repository configuration mode!
2022-01-18 17:39:06.827 INFO  o.s.d.r.config.RepositoryConfigurationDelegate    :132  [32m                  [0;39m Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2022-01-18 17:39:06.948 INFO  o.s.d.r.config.RepositoryConfigurationDelegate    :201  [32m                  [0;39m Finished Spring Data repository scanning in 107 ms. Found 8 JPA repository interfaces.
2022-01-18 17:39:07.138 INFO  o.s.d.r.config.RepositoryConfigurationDelegate    :262  [32m                  [0;39m Multiple Spring Data modules found, entering strict repository configuration mode!
2022-01-18 17:39:07.140 INFO  o.s.d.r.config.RepositoryConfigurationDelegate    :132  [32m                  [0;39m Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2022-01-18 17:39:07.155 INFO  o.s.d.r.c.RepositoryConfigurationExtensionSupport :349  [32m                  [0;39m Spring Data Redis - Could not safely identify store assignment for repository candidate interface com.amber.insect.knowledgebase.repository.CategoryRepository. If you want this repository to be a Redis repository, consider annotating your entities with one of these annotations: org.springframework.data.redis.core.RedisHash (preferred), or consider extending one of the following types with your repository: org.springframework.data.keyvalue.repository.KeyValueRepository.
2022-01-18 17:39:07.156 INFO  o.s.d.r.c.RepositoryConfigurationExtensionSupport :349  [32m                  [0;39m Spring Data Redis - Could not safely identify store assignment for repository candidate interface com.amber.insect.knowledgebase.repository.ContentRepository. If you want this repository to be a Redis repository, consider annotating your entities with one of these annotations: org.springframework.data.redis.core.RedisHash (preferred), or consider extending one of the following types with your repository: org.springframework.data.keyvalue.repository.KeyValueRepository.
2022-01-18 17:39:07.157 INFO  o.s.d.r.c.RepositoryConfigurationExtensionSupport :349  [32m                  [0;39m Spring Data Redis - Could not safely identify store assignment for repository candidate interface com.amber.insect.knowledgebase.repository.ContributeRepository. If you want this repository to be a Redis repository, consider annotating your entities with one of these annotations: org.springframework.data.redis.core.RedisHash (preferred), or consider extending one of the following types with your repository: org.springframework.data.keyvalue.repository.KeyValueRepository.
2022-01-18 17:39:07.157 INFO  o.s.d.r.c.RepositoryConfigurationExtensionSupport :349  [32m                  [0;39m Spring Data Redis - Could not safely identify store assignment for repository candidate interface com.amber.insect.knowledgebase.repository.DocRepository. If you want this repository to be a Redis repository, consider annotating your entities with one of these annotations: org.springframework.data.redis.core.RedisHash (preferred), or consider extending one of the following types with your repository: org.springframework.data.keyvalue.repository.KeyValueRepository.
2022-01-18 17:39:07.157 INFO  o.s.d.r.c.RepositoryConfigurationExtensionSupport :349  [32m                  [0;39m Spring Data Redis - Could not safely identify store assignment for repository candidate interface com.amber.insect.knowledgebase.repository.EbookRepository. If you want this repository to be a Redis repository, consider annotating your entities with one of these annotations: org.springframework.data.redis.core.RedisHash (preferred), or consider extending one of the following types with your repository: org.springframework.data.keyvalue.repository.KeyValueRepository.
2022-01-18 17:39:07.158 INFO  o.s.d.r.c.RepositoryConfigurationExtensionSupport :349  [32m                  [0;39m Spring Data Redis - Could not safely identify store assignment for repository candidate interface com.amber.insect.knowledgebase.repository.EbookSnapshotRepository. If you want this repository to be a Redis repository, consider annotating your entities with one of these annotations: org.springframework.data.redis.core.RedisHash (preferred), or consider extending one of the following types with your repository: org.springframework.data.keyvalue.repository.KeyValueRepository.
2022-01-18 17:39:07.158 INFO  o.s.d.r.c.RepositoryConfigurationExtensionSupport :349  [32m                  [0;39m Spring Data Redis - Could not safely identify store assignment for repository candidate interface com.amber.insect.knowledgebase.repository.TagRepository. If you want this repository to be a Redis repository, consider annotating your entities with one of these annotations: org.springframework.data.redis.core.RedisHash (preferred), or consider extending one of the following types with your repository: org.springframework.data.keyvalue.repository.KeyValueRepository.
2022-01-18 17:39:07.159 INFO  o.s.d.r.c.RepositoryConfigurationExtensionSupport :349  [32m                  [0;39m Spring Data Redis - Could not safely identify store assignment for repository candidate interface com.amber.insect.knowledgebase.repository.UserRepository. If you want this repository to be a Redis repository, consider annotating your entities with one of these annotations: org.springframework.data.redis.core.RedisHash (preferred), or consider extending one of the following types with your repository: org.springframework.data.keyvalue.repository.KeyValueRepository.
2022-01-18 17:39:07.159 INFO  o.s.d.r.config.RepositoryConfigurationDelegate    :201  [32m                  [0;39m Finished Spring Data repository scanning in 9 ms. Found 0 Redis repository interfaces.
2022-01-18 17:39:08.110 INFO  o.s.boot.web.embedded.tomcat.TomcatWebServer      :108  [32m                  [0;39m Tomcat initialized with port(s): 8880 (http)
2022-01-18 17:39:08.119 INFO  org.apache.coyote.http11.Http11NioProtocol        :173  [32m                  [0;39m Initializing ProtocolHandler ["http-nio-8880"]
2022-01-18 17:39:08.120 INFO  org.apache.catalina.core.StandardService          :173  [32m                  [0;39m Starting service [Tomcat]
2022-01-18 17:39:08.120 INFO  org.apache.catalina.core.StandardEngine           :173  [32m                  [0;39m Starting Servlet engine: [Apache Tomcat/9.0.56]
2022-01-18 17:39:08.199 INFO  o.a.c.core.ContainerBase.[Tomcat].[localhost].[/] :173  [32m                  [0;39m Initializing Spring embedded WebApplicationContext
2022-01-18 17:39:08.199 INFO  o.s.b.w.s.c.ServletWebServerApplicationContext    :290  [32m                  [0;39m Root WebApplicationContext: initialization completed in 4469 ms
2022-01-18 17:39:08.520 INFO  org.hibernate.jpa.internal.util.LogHelper         :31   [32m                  [0;39m HHH000204: Processing PersistenceUnitInfo [name: default]
2022-01-18 17:39:08.570 INFO  org.hibernate.Version                             :44   [32m                  [0;39m HHH000412: Hibernate ORM core version 5.6.3.Final
2022-01-18 17:39:10.819 INFO  org.hibernate.annotations.common.Version          :56   [32m                  [0;39m HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-01-18 17:39:10.972 INFO  com.alibaba.druid.pool.DruidDataSource            :1003 [32m                  [0;39m {dataSource-1} inited
2022-01-18 17:39:16.398 ERROR com.alibaba.druid.pool.DruidDataSource            :2699 [32m                  [0;39m create connection SQLException, url: jdbc:mysql://localhost/knowledge-base?characterEncoding=UTF8&autoReconnect=true&serverTimezone=Asia/Shanghai&allowMultiQueries=true, errorCode 0, state 08001
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:906)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:831)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1596)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1662)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2697)
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'localhost' (using password: YES)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:792)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:717)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:685)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:134)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:469)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:174)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1348)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:157)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:850)
	... 7 common frames omitted
2022-01-18 17:39:21.113 ERROR com.alibaba.druid.pool.DruidDataSource            :2699 [32m                  [0;39m create connection SQLException, url: jdbc:mysql://localhost/knowledge-base?characterEncoding=UTF8&autoReconnect=true&serverTimezone=Asia/Shanghai&allowMultiQueries=true, errorCode 0, state 08001
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:906)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:831)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1596)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1662)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2697)
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'localhost' (using password: YES)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:792)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:717)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:685)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:134)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:469)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:174)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1348)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:157)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:850)
	... 7 common frames omitted
2022-01-18 17:39:21.113 INFO  com.alibaba.druid.pool.DruidAbstractDataSource    :1743 [32m                  [0;39m {dataSource-1} failContinuous is true
2022-01-18 17:39:27.403 ERROR com.alibaba.druid.pool.DruidDataSource            :2699 [32m                  [0;39m create connection SQLException, url: jdbc:mysql://localhost/knowledge-base?characterEncoding=UTF8&autoReconnect=true&serverTimezone=Asia/Shanghai&allowMultiQueries=true, errorCode 0, state 08001
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:906)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:831)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1596)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1662)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2697)
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'localhost' (using password: YES)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:792)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:717)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:685)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:134)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:469)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:174)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1348)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:157)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:850)
	... 7 common frames omitted
2022-01-18 17:39:32.557 ERROR com.alibaba.druid.pool.DruidDataSource            :2699 [32m                  [0;39m create connection SQLException, url: jdbc:mysql://localhost/knowledge-base?characterEncoding=UTF8&autoReconnect=true&serverTimezone=Asia/Shanghai&allowMultiQueries=true, errorCode 0, state 08001
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:906)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:831)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1596)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1662)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2697)
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'localhost' (using password: YES)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:792)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:717)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:685)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:134)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:469)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:174)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1348)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:157)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:850)
	... 7 common frames omitted
2022-01-18 17:39:37.801 ERROR com.alibaba.druid.pool.DruidDataSource            :2699 [32m                  [0;39m create connection SQLException, url: jdbc:mysql://localhost/knowledge-base?characterEncoding=UTF8&autoReconnect=true&serverTimezone=Asia/Shanghai&allowMultiQueries=true, errorCode 0, state 08001
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:906)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:831)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1596)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1662)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2697)
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'localhost' (using password: YES)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:792)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:717)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:685)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:134)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:469)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:174)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1348)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:157)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:850)
	... 7 common frames omitted
2022-01-18 17:39:44.107 ERROR com.alibaba.druid.pool.DruidDataSource            :2699 [32m                  [0;39m create connection SQLException, url: jdbc:mysql://localhost/knowledge-base?characterEncoding=UTF8&autoReconnect=true&serverTimezone=Asia/Shanghai&allowMultiQueries=true, errorCode 0, state 08001
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:906)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:831)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1596)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1662)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2697)
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'localhost' (using password: YES)
	at sun.reflect.GeneratedConstructorAccessor53.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:792)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:717)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:685)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:134)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:469)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:174)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1348)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:157)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:850)
	... 7 common frames omitted
2022-01-18 17:39:49.484 ERROR com.alibaba.druid.pool.DruidDataSource            :2699 [32m                  [0;39m create connection SQLException, url: jdbc:mysql://localhost/knowledge-base?characterEncoding=UTF8&autoReconnect=true&serverTimezone=Asia/Shanghai&allowMultiQueries=true, errorCode 0, state 08001
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:906)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:831)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1596)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1662)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2697)
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'localhost' (using password: YES)
	at sun.reflect.GeneratedConstructorAccessor53.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:792)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:717)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:685)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:134)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:469)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:174)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1348)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:157)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:850)
	... 7 common frames omitted
2022-01-18 17:39:54.322 ERROR com.alibaba.druid.pool.DruidDataSource            :2699 [32m                  [0;39m create connection SQLException, url: jdbc:mysql://localhost/knowledge-base?characterEncoding=UTF8&autoReconnect=true&serverTimezone=Asia/Shanghai&allowMultiQueries=true, errorCode 0, state 08001
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:906)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:831)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1596)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1662)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2697)
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'localhost' (using password: YES)
	at sun.reflect.GeneratedConstructorAccessor53.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:792)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:717)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:685)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:134)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:469)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:174)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1348)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:157)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:850)
	... 7 common frames omitted
2022-01-18 17:40:01.099 ERROR com.alibaba.druid.pool.DruidDataSource            :2699 [32m                  [0;39m create connection SQLException, url: jdbc:mysql://localhost/knowledge-base?characterEncoding=UTF8&autoReconnect=true&serverTimezone=Asia/Shanghai&allowMultiQueries=true, errorCode 0, state 08001
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:906)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:831)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1596)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1662)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2697)
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'localhost' (using password: YES)
	at sun.reflect.GeneratedConstructorAccessor53.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:792)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:717)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:685)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:134)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:469)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:174)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1348)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:157)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:850)
	... 7 common frames omitted
2022-01-18 17:40:06.793 ERROR com.alibaba.druid.pool.DruidDataSource            :2699 [32m                  [0;39m create connection SQLException, url: jdbc:mysql://localhost/knowledge-base?characterEncoding=UTF8&autoReconnect=true&serverTimezone=Asia/Shanghai&allowMultiQueries=true, errorCode 0, state 08001
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:906)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:831)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1596)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1662)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2697)
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'localhost' (using password: YES)
	at sun.reflect.GeneratedConstructorAccessor53.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:792)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:717)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:685)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:134)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:469)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:174)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1348)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:157)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:850)
	... 7 common frames omitted
2022-01-18 17:40:13.547 ERROR com.alibaba.druid.pool.DruidDataSource            :2699 [32m                  [0;39m create connection SQLException, url: jdbc:mysql://localhost/knowledge-base?characterEncoding=UTF8&autoReconnect=true&serverTimezone=Asia/Shanghai&allowMultiQueries=true, errorCode 0, state 08001
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:906)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:831)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1596)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1662)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2697)
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'localhost' (using password: YES)
	at sun.reflect.GeneratedConstructorAccessor53.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:792)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:717)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:685)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:134)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:469)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:174)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1348)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:157)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:850)
	... 7 common frames omitted
